pkg/gatherers/clusterconfig/operators_pods_and_events.go

```diff
@@ -313,6 +309,37 @@ func getContainerLogs(ctx context.Context,
 	return records
 }
 
+func getLogWithStacktracing(logString string) (string, bool) {
+	reader := strings.NewReader(logString)
+	scanner := bufio.NewScanner(reader)
+	scanner.Split(bufio.ScanLines)
+
+	var log []string
+	var found bool
+	for scanner.Scan() {
+		line := scanner.Text()
+		// if it already found the stack we just want to keep add the log till the end
+		if found {
+			log = append(log, line)
+			continue
+		}
+		// tries to find the stack message at the given line and add it to the log
+		if found = stackTraceRegex.MatchString(line); found {
+			log = append(log, line)
+			continue
+		}
+		// if it didn't find the line yet, keep add it as offset limiting by the arbitrary
+		// offset limit value
+		log = append(log, line)
+		if int64(len(log)) > logLinesOffset {
+			log[0] = "" // to avoid memory leaks
+			log = log[1:]
+		}
+	}
+
+	return strings.Join(log, "\n"), found
```

Comment:
If we don't find any stack tracing message... I think that would be nice to get a smaller log... What do you think?